vision/segmentation/radius_clustering.go

```diff
@@ -18,6 +18,8 @@ import (
 type RadiusClusteringConfig struct {
 	resource.TriviallyValidateConfig
 	MinPtsInPlane      int     `json:"min_points_in_plane"`
+	MaxDistFromPlane   float64 `json:"max_dist_from_plane"`
+	MaxAngleOfPlane    float64 `json:"max_angle_of_plane"`
```

Comment:
We should send out a scope update for any config changes.

Reply:
Sent the email

Reply:
thank you!